docs/scheduler.rst

```diff
@@ -32,161 +30,31 @@ Airflow production environment. To kick it off, all you need to do is
 execute ``airflow scheduler``. It will use the configuration specified in
 ``airflow.cfg``.
 
-Note that if you run a DAG on a ``schedule_interval`` of one day,
-the run stamped ``2016-01-01`` will be triggered soon after ``2016-01-01T23:59``.
-In other words, the job instance is started once the period it covers
-has ended.
-
-**Let's Repeat That** The scheduler runs your job one ``schedule_interval`` AFTER the
-start date, at the END of the period.
-
-The scheduler starts an instance of the executor specified in the your
-``airflow.cfg``. If it happens to be the :class:`airflow.contrib.executors.local_executor.LocalExecutor`, tasks will be
-executed as subprocesses; in the case of :class:`airflow.executors.celery_executor.CeleryExecutor`, and :class:`airflow.executors.dask_executor.DaskExecutor` tasks are executed remotely.
+The scheduler starts an instance of an :doc:`Executor </executor/index>`. 
```

Comment:
```suggestion
The scheduler uses the configured :doc:`Executor </executor/index>` to run tasks that are ready.
```

Only subtly different here, but I feel that users of airflow don't need care that much about details of class instantiation :)